Pawns/DeusEx/Classes/NSFCloneAugStealth1NametagCarcass.uc

```diff
@@ -0,0 +1,22 @@
+//=============================================================================
+// NSFCloneAugStealth1Carcass.
+//=============================================================================
+class NSFCloneAugStealth1Carcass extends #var(DeusExPrefix)Carcass;
+
+defaultproperties
+{
+    Fatness=130
```

Comment:
pretty sure Fatness of 127 is the middle value, this value of 130 probably came from the tough guy?